### PR TITLE
Bug 1836374: OWNERS: add Dan Mace and Suresh Kolichala 

### PR DESCRIPTION
**Pull request by hexfusion**
Signed-off-by: Sam Batschelet <sbatsche@redhat.com>

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -2,13 +2,14 @@ reviewers:
...
   - alaypatel07
+  - ironcladlou
   - mfojtik
   - soltysh
   - sttts
 approvers:
   - deads2k
   - hexfusion
-  - alaypatel07
+  - ironcladlou
```
Comment (ironcladlou) [inline OWNERS:12]: I think Alay probably has more claim to approver than I do at this point

File: OWNERS [hunk 2]
```diff
@@ -1,15 +1,13 @@
 reviewers:
+  - alaypatel07
```
Comment (deads2k) [inline OWNERS:2]: you don't want him here.  He won't be actively reviewing, so it can skew assignments.

File: OWNERS [hunk 3]
```diff
@@ -1,15 +1,13 @@
...
   - deads2k
```
Comment (deads2k) [inline OWNERS:3]: pull me from the regular reviewers. I'll come in for approval or if you need me.

**Comment by deads2k** [comment]
/lgtm

welcome to the wild west :)

**Pull request merged**